data/tlg002/tlg0559.tlg002.xml

```diff
@@ -0,0 +1,1942 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader>
+<fileDesc>
+<titleStmt>
+ <title xml:lang="lat">Heronis Alexandrini opera quae supersunt omnia, vol. 1: Pneumatica et automata</title>
+<author>Hero of Alexandria</author>	
+ <editor>Wilhelm Schmidt</editor>
+<sponsor>Harvard College Library</sponsor>
```

Comment:
The section from `<sponsor>` to the end of the last `<respStmt>` should be replaced by:

```
<funder>Harvard Library Arcadia Fund</funder>
    <principal>Gregory Crane</principal>
    <respStmt>
     <persName xml:id="DDD">Digital Divide Data</persName>
     <resp>Corrected and encoded the text</resp>
    </respStmt>
    <respStmt>
     <persName>Gregory Crane</persName>
     <resp>Editor-in-Chief, Perseus Digital Library</resp>
    </respStmt>
    <respStmt>
     <persName>Matt Munson</persName>
     <resp>Project Manager (University of Leipzig), 2016 - present</resp>
    </respStmt>
    <respStmt>
     <persName>Annette Gessner</persName>
     <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
    </respStmt>
    <respStmt>
     <persName>Thibault Clérice</persName>
     <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
    </respStmt>
    <respStmt>
     <persName>Bruce Robertson</persName>
     <resp>Technical Advisor (Mount Allison University)</resp>
    </respStmt>
```